netmanager/src/views/layouts/common/Sidebar/Sidebar.js

```diff
@@ -104,6 +104,11 @@ const allMainPages = [
     href: "/manager",
     icon: <ManageIcon />,
     collapse: true,
+    nested: true,
+    nestItems: [
+      { title: "Device Map", href: "/manager/map" },
```

Comment:
Could we rename these to: 

1. **Network map** and 
2. **Network statistics**?